src/content/resources/javascript/es6/arrow-functions.md

```diff
@@ -1,11 +1,59 @@
 ---
 authors:
   - "veksen#1060"
+  - "t0m#5956"
 created_at: 2019/10/08
-updated_at: 2019/10/08
+updated_at: 2019/10/13
 title: Arrow functions
 ---
+## Arrow Function Syntax
 
-## Arrow functions
+Arrow functions are a new way to declare functions within ES6. Arrow functions allow for a shorter syntax. For example, below is a regular function, as usually seen before ES6.
+```javascript
+const sayHello = function() {
+  console.log('hello');
+}
+```
+The same function can be rewritten with arrow functions, as seen below.
+```javascript
+const sayHello = () => {
+  console.log('hello');
+}
+```
+Both of the above functions have no parameters, but if you want to pass parameters into the function with the arrow syntax, you can do the following, placing the parameters within the brackets of the arrow function.
+```javascript
+const sayHello = (nameOne,nameTwo) => {
+  console.log(nameOne,'says hello to',nameTwo);
+}
+```
+If you want to pass only one parameter, you don't need the brackets around the parameters, and can do the following:
+```javascript
+const sayHello = name => {
+  console.log('hello',name);
+}
+```
+Another interesting feature that also reduces the syntax within arrow functions, is that if a function only returns an expression using one line of code, you can omit the ``return`` keyword and the curly braces. Be aware that in multiple line arrow functions, the return keyword still needs to be used. The following function returns "hello":
```

Comment:
This second sentence is not entirely accurate. The following is valid :

```
() =>
  "hello"
```

Reply:
This still needs resolved

Reply:
How would you prefer to word it? "Be aware that in multiple expression arrow functions, the return keyword still needs to be used. "?

Reply:
Multiple-expression arrow functions sounds a bit confusing, how about expanded arrow functions?

Reply:
No, the only place where the return keyword is needed, is when you use braces (`{ return "something"}`). When braces are omitted, the return is implicit.